packaging/s3plugin_lib.py

```diff
@@ -32,3 +32,29 @@ def make_arbitrary_file(f_name, f_size, buffer_size=32*1024*1024):
             out.write(bytearray(buffer))
 
             bytes_written += to_write
+
+def crc64_nvme(data: bytes, initial_crc = 0):
+    """
+    Calculates the CRC64-NVMe checksum for the given data.
+
+    Args:
+        data: The bytes object for which to calculate the CRC.
+        initial_crc: The starting CRC value for incremental calculations.
+                     Defaults to 0 for a fresh calculation.
+
+    Returns:
+        The 64-bit unsigned integer representing the CRC64-NVMe checksum.
+    """
+    polynomial = 0x9a6c93291d196501
+    polynomial = 0x9a6c9329ac4bc9b5
```

Comment:
Assuming we don't need that first one?

Also, a little confused by the name on this one (how is this a polynomial?), but maybe that's what it's called in the CRC64-NVMe checksum calculation algorithm?

Reply:
Ah, thanks